mahta/CSS/CSS-box-model/styles4.css

```diff
@@ -0,0 +1,38 @@
+.container {
+  display: grid;
```

Comment:
I didn't meant using `grid` layout. You could achieve the same using margins. Anyways, `grid` is far more powerful. Nice job reading about it! 💪
(This comment is just to consider your extra efforts, there is no need to change anything.)

Reply:
Thanks a lot! I'll try to code using margins as well, since I didn't find it functioning at first.

Reply:
There is no much point doing it. You know the box model now, and that's enough.